templates/community-viz/README.md

```diff
@@ -25,38 +25,38 @@ directory.
 
 ## Using this template
 
-To create a new community visualization using this template, run the command
+To create a new community visualization based on this template, run the command
 
 ```bash
 npx @google/dscc-gen --project_choice community-viz
 ```
 
 At the end of the setup flow, you will have a minimal working visualization and
-have set GCS buckets for a dev and prod version.
+have set GCS buckets for a dev and prod version. Edit `src/index.js` to see changes.
 
-### Writing the viz
+### Update your local data
 
-1. Define the dimensions and metrics your visualization requires in
+1. Update the dimensions and metrics your visualization requires in
    `src/index.json`
-2. Run the commands `npm run build:dev and; npm run push:dev` to build and
+1. Run the commands `npm run updateMessage -- --format object` to build and
    deploy your visualization to your "dev" bucket.
-3. [Create a new report][datastudio] and connect to the dataset you want to use
+1. [Create a new report][datastudio] and connect to the dataset you want to use
    for your sample message. I'd recommend using a subset of your data, to keep
    the size of the message small.
-4. Use your "dev bucket" to add this visualization to your report. It will
+1. Use your "dev bucket" to add this visualization to your report. It will
```

Comment:
we can replace "dev bucket" here with the values that they put in the prompts which would make this readme easier to follow.

Reply:
I'll track this in an issue, because there's another place I want to do that.